src/plugins/list/README.md

```diff
@@ -48,6 +48,23 @@ A list of set-placements for the plugin. Same for "get" and "error"
 
 Plugin specific config.
 
+## Exported functions
```

Comment:
Please use [title case](https://www.grammar-monster.com/lessons/capital_letters_title_case.htm) for all headings.

Reply:
I think this needs to be added to doc/CODING.md?